File: test/jskills/trueskill/TwoPlayerTrueSkillCalculatorTest.java
package jskills.trueskill;

import org.testng.annotations.Test;

public class TwoPlayerTrueSkillCalculatorTest {
    @Test
    public void TwoPlayerTrueSkillCalculatorTests() {
        TwoPlayerTrueSkillCalculator calculator = new TwoPlayerTrueSkillCalculator();

        // We only support two players
        TrueSkillCalculatorTests.TestAllTwoPlayerScenarios(calculator);

        // TODO: Assert failures for larger teams
    }    
}

File: src/test/scala/jskills/trueskill/TwoPlayerTrueSkillCalculatorTest.scala
package jskills.trueskill

import org.junit.Before
import org.junit.Test

class TwoPlayerTrueSkillCalculatorTest extends TrueSkillCalculatorTestUtils {

  var calculator: TwoPlayerTrueSkillCalculator = null

  @Before def setup() {
    calculator = new TwoPlayerTrueSkillCalculator()
  }

  @Test def testAllTwoPlayerScenarios() {
    // We only support two players
    TestAllTwoPlayerScenarios(calculator)
  }

  // TODO: Assert failures for larger teams
}